wsmaster/che-core-sql-schema/src/main/resources/che-schema/7.11.0/1__update_inconsistent_stopped_workspace_activities.sql

```diff
@@ -0,0 +1,15 @@
+--
+-- Copyright (c) 2012-2020 Red Hat, Inc.
+-- This program and the accompanying materials are made
+-- available under the terms of the Eclipse Public License 2.0
+-- which is available at https://www.eclipse.org/legal/epl-2.0/
+--
+-- SPDX-License-Identifier: EPL-2.0
+--
+-- Contributors:
+--   Red Hat, Inc. - initial API and implementation
+--
+
+UPDATE che_workspace_activity
+    SET last_stopped = 0
+    WHERE status = 'STOPPED' AND last_stopped IS NULL;
```

Comment:
Can we alter the table after that and make `last_stopped NOT NULL`? WDYT?

Reply:
Having `last_stopped NOT NULL` will cause failure when updating activity that was not created before -> https://github.com/eclipse/che/blob/master/wsmaster/che-core-api-workspace-activity/src/main/java/org/eclipse/che/api/workspace/activity/JpaWorkspaceActivityDao.java#L300 Such activity does not even have a status. I don't know when this can happen. Do you think it's worth going to that rabbit hole ?